utils/bot.js

```diff
@@ -1,17 +1,17 @@
 const { avatarFormat } = require("./user");
 
 function partialBotObject(bot) {
-    return {
-        tags: bot.details.tags,
-        avatar: avatarFormat(bot),
-        name: bot.username,
-        id: bot.id || bot._id,
-        status: bot.status,
-        description: bot.details.shortDescription,
-        votes: bot.votes.current
-    }
+  return {
+    tags: bot.details.tags,
+    avatar: avatarFormat(bot),
+    name: bot.username,
+    id: bot.id || bot._id,
+    status: bot.status,
+    description: bot.details.shortDescription,
+    votes: bot.votes ? bot.votes.current : -1,
```

Comment:
I've added my bot, but when i open home page, throw error in these line 🤔 
Simple fix :yey: